docs/notebooks/sdba-advanced.ipynb

```diff
@@ -38,11 +38,11 @@
    "source": [
     "## LOESS smoothing and detrending\n",
     "\n",
-    "As described in Cleveland (1979), locally weighted linear regressions are multiple regression methods using a nearest-neighbor approach. Instead of using all data points to compute a linear or polynomial regression, LOESS algorithms compute a local regression for each point in the dataset, using only the k-nearest neighbors as selected by a weighting function. This weighting function must fulfill some strict requirements, see the doc of `xclim.sdba.loess.loess_smoothing` for more details.\n",
+    "As described in Cleveland (1979), locally weighted linear regressions are multiple regression methods using a nearest-neighbour approach. Instead of using all data points to compute a linear or polynomial regression, LOESS algorithms compute a local regression for each point in the dataset, using only the k-nearest neighbours as selected by a weighting function. This weighting function must fulfill some strict requirements, see the doc of `xclim.sdba.loess.loess_smoothing` for more details.\n",
```

Comment:
Great question. I think we run into this problem often, since Canadian English can sometimes be an unholy amalgam of the two at times ("Metric or imperial? Why not use both?").

Let's go with British. Will fix.

Reply:
Hmm, I thought Canadian was more aligned with British, but in the _common use_ it's a mix of both? Was not aware it's an official mix haha. Do what you think is best